### PR TITLE
Sarahsimionescu/tech 25 faq section

### PR DESCRIPTION
**Pull request by sarahsimionescu**

### CODE REVIEW

**Comment by sarahsimionescu** [comment]
Some of the checks don't work after I installed react - plz help!

**Comment by sarahsimionescu** [comment]
My code is not pretty enough - plz help!

**Review by err53 — APPROVED** [review]
component lgtm! we'll need to add some global spacing between the components but the component by itself is good!

File: src/components/FAQ.astro [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  {
+    question: "Quis autem vel eum iure reprehenderit?",
+    answer:
+      "Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto.",
+  },
+];
+---
+
+<div class="container flex max-w-3xl flex-col items-center text-white">
+  <h1 class="mb-10 text-center font-display text-6xl font-bold">
```
Comment (err53) [inline src/components/FAQ.astro]: Wait, please don't use h1 for anything other than the page title. This is very bad for accessibility.

**Review by err53 — CHANGES_REQUESTED** [review]

File: src/components/FAQ.astro (hunk 1 above)
Comment (sarahsimionescu) [reply src/components/FAQ.astro]: Will fix!

**Review by err53 — APPROVED** [review]

**Pull request merged**